apps/emqx_resource/include/emqx_resource.hrl

```diff
@@ -159,4 +159,12 @@
 -define(TEST_ID_PREFIX, "t_probe_").
 -define(RES_METRICS, resource_metrics).
 
+-define(LOG_LEVEL(_L_),
+    case _L_ of
+        true -> info;
+        false -> warning
+    end
+).
```

Comment:
Seems unused?  :scissors:

Reply:
it‘s used in https://github.com/emqx/emqx/pull/13449， Just cherry-pick from that commit.